### ISSUE TITLE
trakt sign-in issues

### ISSUE THREAD

**Issue by UweTrottmann** [UweTrottmann/SeriesGuide#429]
Collecting affected devices to maybe find a pattern.

This assumes that the user tried to clear the SeriesGuide cache and changed the password but still failed.

**Affected devices**
- Samsung Tab 3 8.0, [422 Unprocessable entity](http://tools.ietf.org/html/rfc4918#section-11.2) “I darf you very gerp” *odd, as only used for WebDAV*

**Comment by gabotronix** [comment]
LG G Pad 8.3 V500
Android Version 4.4.2 (rooted)
Trakt Error 422, the change you wanted was rejected

**Comment by UweTrottmann** [comment]
@gabotronix You should have a look at the updated help section at http://seriesgui.de/help#trakt. It's likely that a proxy or VPN has caused your issues.

Closing this as all remaining issues were traced back to VPN/proxy settings.

**Comment by gabotronix** [comment]
Thanks, @UweTrottmann. I'm not connected any VPN or Proxy from my device. However, I saw  *ad-blocking* from the help section you mentioned. I do have **Adblock Plus** for Android installed. Adblock Plus resides as a background app that filters web traffic from all sources. I access it through the system tray. I temporarily disabled *Filtering* from ABP, and I was able to log in to Trakt from SeriesGuide afterwards. Enabling ABP Filtering again after login seems to not affect SeriesGuide. So all good now.
